wordpress-mobile/gutenberg-mobile
# Update usage of useInnerBlocksProps API

Updates the Tiled Gallery feature branch to fix an error caused by the `useInnerBlocksProps` API changing name.

Related: 
- https://github.com/Automattic/jetpack/pull/21625

### To test

1. Run the packager by executing `npm run start:reset` in one terminal window
2. Run the demo app for iOS in another terminal window, e.g `npm run core ios`
3. Verify the block can be inserted via the block inserter, the block inserter search, and the slash inserter
4. Run the demo app for Android, `npm run core android`
5. Repeat step 3

PR submission checklist:
